hbase/getmetrics_hbase.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+
+import linecache
+import json
+import csv
+from subprocess import Popen, PIPE, STDOUT
+import time
+import os
+from optparse import OptionParser
+import multiprocessing
+import socket
+import fileinput
+from itertools import izip
+import commands
+
+# This script gathers hbase operation execution time and add to csv file
+
+def getindex(col_name): # need to generate the group id from a range
+#    return random.randint(3000,10000)
+#    if col_name == "alter":
+#        return 21001
+#    if col_name == "create":
+#        return 21002
+#    if col_name == "describe":
+#        return 21003
+#    if col_name == "disable":
+#        return 21004
+#    if col_name == "disable_all":
+#        return 21005
+#    if col_name == "is_disabled":
+#        return 21006
+#    if col_name == "drop":
+#        return 21007
+#    if col_name == "drop_all":
+#        return 21008
+#    if col_name == "enable":
+#        return 21009
+#    if col_name == "enable_all":
+#        return 21010
+#    if col_name == "is_enabled":
+#        return 21011
+#    if col_name == "exists":
+#        return 21012
+#    if col_name == "list":
+#        return 21013
+#    if col_name == "count":
+#        return 21014
+#    if col_name == "delete":
+#        return 21015
+#    if col_name == "delete_all":
+#        return 21016
+    if col_name == "get":
+        return 21017
+#    if col_name == "get_Counter":
+#        return 21018
+#    if col_name == "incr":
+#        return 21019
+    if col_name == "put":
+        return 21020
+#    if col_name == "scan":
+#        return 21021
+#    if col_name == "truncate":
+#        return 21022
+
+def listtocsv(lists):
+    log = ''
+    for i in range(0,len(lists)):
+        log = log + str(lists[i])
+        if(i+1 != len(lists)):
+		log = log + ','
+    hbase_execution_time.write("%s\n"%(log))
+
+
+usage = "Usage: %prog [options]"
+parser = OptionParser(usage=usage)
```

Comment:
Use a seperate function for config parsing. Use example(https://github.com/insightfinder/InsightAgent/blob/kafka_chunking/kafka/getmetrics_kafka.py)